workers/social/lib/social/models/group/groupdata.coffee

```diff
@@ -77,22 +77,25 @@ module.exports = class JGroupData extends Model
       payload = if data then getAt data.payload, path else null
       return callback null, payload
 
+
   # see docs on JGroup::modifyData
-  @modifyData : (slug, data, callback) ->
-    return callback new Error 'slug is required' unless slug
+  @modifyData = (slug, data, callback) ->
+    return callback new Error 'slug is required'  unless slug
 
     # it's only allowed to change followings
-    whitelist  = ['github.organizationToken', 'test_key__' ]
+    allowedPaths = [ 'github.organizationToken', 'test_key__' ]
 
     # handle $set and $unset cases in one
-    operation  = { $set: {}, $unset: {} }
-
-    for item in whitelist
-      key = "payload.#{item}"
-      val = getAt data, item
-      if val
-      then operation.$set[key]   = val
-      else operation.$unset[key] = ''
+    operation = {}
+
+    allowedPaths.forEach (path) ->
+      key = "payload.#{path}"
+      if val = data[path]
```

Comment:
yep, it was using `getAt` for old implementation, but for now it's checking for existing keys on a given object which doesn't require `getAt` at all.